yb-voyager/src/dbzm/yb_cdc_client.go

```diff
@@ -81,7 +82,7 @@ func (ybc *YugabyteDBCDCClient) GenerateAndStoreStreamID() (string, error) {
 	args := fmt.Sprintf("-create -master_addresses %s -table_name %s -db_name %s ", ybc.ybMasterNodes, ybc.tableName, ybc.dbName)
 
 	if ybc.sslRootCert != "" {
-		args += fmt.Sprintf(" -ssl_cert_file %s", ybc.sslRootCert)
+		args += fmt.Sprintf(" -ssl_cert_file '%s'", ybc.sslRootCert)
```

Comment:
what is the context for this change?

Reply:
I hit this problem when I was trying out with the root cert: If the root cert file name has special characters or spaces, this command will error out if quotes are not present around the file path. 
ex. root-cert  `/<path>/root (2).crt`
This might not be really possible in real scenario, but I thought its better to fix it.